app/views/layouts/application.html.erb

```diff
@@ -46,7 +46,7 @@
     <%= render "shared/header" %>
 
     <div class="govuk-width-container">
-      <% if user_signed_in? %>
+      <% if user_signed_in? && request.path != "/users/invitations/invite_second_admin" %>
```

Comment:
Not too sure about this comparison with a specific URL. We could have a method like 'show_navigation_bars' or whatever in the application_controller and overload it as necessary in the right controller